source/gui/settingsDialogs.py

```diff
@@ -2566,6 +2566,13 @@ def __init__(self, parent):
 		self.UIAInMSWordCheckBox.SetValue(config.conf["UIA"]["useInMSWordWhenAvailable"])
 		self.UIAInMSWordCheckBox.defaultValue = self._getDefaultValue(["UIA", "useInMSWordWhenAvailable"])
 
+		# Translators: This is the label for a checkbox in the
+		#  Advanced settings panel.
+		label = _("Use UI Automation to access Microsoft &Excel spreadsheet controls when available")
+		self.UIAInMSExcelCheckBox = UIAGroup.addItem(wx.CheckBox(self, label=label))

```

Comment:
Please do not forget context help, as mentionned by @lukaszgo1.

```suggestion
		self.UIAInMSExcelCheckBox = UIAGroup.addItem(wx.CheckBox(self, label=label))
		self.bindHelpEvent("UseUiaForExcel", self.UIAInMSExcelCheckBox)

```

PS: The first line did not change, but I do not know how to make a suggestion to insert one line without modifying the others in GitHub.